BE/PoroDev.DatabaseService/Consumers/StorageServiceConsumer/FileUploadConsumer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using AutoMapper;
+using MassTransit;
+using MongoDB.Bson;
+using PoroDev.Common.Contracts;
+using PoroDev.Common.Contracts.StorageService;
+using PoroDev.Common.Contracts.StorageService.UploadFile;
+using PoroDev.Common.Models.StorageModels.Data;
+using PoroDev.DatabaseService.Repositories.Contracts;
+
+namespace PoroDev.DatabaseService.Consumers.StorageServiceConsumer
+{
+    public class FileUploadConsumer : IConsumer<FileUploadRequestServiceToDatabase>
+    {
+        private IUnitOfWork _unitOfWork;
+        private IMapper _mapper;
+        private IFileRepository _fileRepository;
+  
+
+        public FileUploadConsumer(IUnitOfWork unitOfWork, IMapper mapper, IFileRepository fileRepository)
+        {
+            _unitOfWork = unitOfWork;
+            _mapper = mapper;
+            _fileRepository = fileRepository;
+        }
+
+        public async Task Consume(ConsumeContext<FileUploadRequestServiceToDatabase> context)
+        {
+                   
+            ObjectId id = await _fileRepository.UploadFile(context.Message.FileName, context.Message.File, context.Message.ContentType, context.Message.UserId);
+
+            FileUploadModel model = new(context.Message.FileName, context.Message.File, context.Message.ContentType, context.Message.UserId);
```

Comment:
Lets be consistent here, either you use 
TypeOfClass variableName = new();
or 
var variableName = new TypeOfClass();

please compare 31 line with 32 line.